src/page-modules/contact/ticket-control/complaint/complaintFormMachine.ts

```diff
@@ -0,0 +1,398 @@
+import { and, assign, or, setup } from 'xstate';
+
+export const formMachine = setup({
+  types: {
+    context: {} as {
+      subject: string;
+      feeNumber: string;
+      registeredMobile: string | undefined;
+      customerNumber: string | undefined;
+      travelcard: string | undefined;
+      feedback: string;
+      firstname: string;
+      lastname: string;
+      address: string;
+      postalCode: string;
+      city: string;
+      email: string;
+      phonenumber: string;
+      bankAccount: string;
+      iban: string;
+      swift: string;
+      ticketStorageMode: string | undefined;
```

Comment:
Note: Need to verify these against the API when it's done. No need for change now